client/components/TimeAgo/TimeAgo.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 	date: number | string | Date;
 	useDateCutoffDays?: number;
 	className?: string;
-	now?: number | string | Date;
+	now?: number;
```

Comment:
now can only be a number type, and if you go up the call stack youll see a number is only ever passed into `now`